gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/test/java/io/gravitee/rest/api/service/v4/impl/PrimaryOwnerServiceImplTest.java

```diff
@@ -66,14 +68,26 @@ public class PrimaryOwnerServiceImplTest {
     @Mock
     private RoleService roleService;
 
+    @Mock
+    private ApiPrimaryOwnerDomainService primaryOwnerDomainService;
+
     private static final ExecutionContext EXECUTION_CONTEXT = new ExecutionContext("TEST", "TEST");
 
     @Before
     public void setUp() {
-        this.primaryOwnerService = new PrimaryOwnerServiceImpl(userService, membershipService, groupService, parameterService, roleService);
+        this.primaryOwnerService =
+            new PrimaryOwnerServiceImpl(
+                userService,
+                membershipService,
+                groupService,
+                parameterService,
+                roleService,
+                primaryOwnerDomainService
+            );
     }
 
     @Test
+    @Ignore
```

Comment:
Do we want to keep the test as it won't work anymore, or should we modify the mock to correct it?